src/index.ts

```diff
@@ -65,6 +65,9 @@ export {
   setGraphqlEndpoints,
   setArchiveGraphqlEndpoint,
   sendZkapp,
+  KeyPair,
```

Comment:
What do you think about providing the new functions under a namespace, similarly to `Mina` for everything Mina/tx related? Something like `Lightnet.KeyPair`, `Lightnet.acquireKeyPair`, ...

Reply:
+1, I don't want more top-level exports (we already have way too many)